src/game/object_list_processor.c

```diff
@@ -559,7 +563,13 @@ void update_non_terrain_objects(void) {
 
     s32 i = 2;
     while ((listIndex = sObjectListUpdateOrder[i]) != -1) {
+        if (listIndex == OBJ_LIST_PLAYER) {
+            fast_profiler_update(PROFILER_TIME_BEHAVIOR1);
+        }
         gObjectCounter += update_objects_in_list(&gObjectLists[listIndex]);
+        if (listIndex == OBJ_LIST_PLAYER) {
+            fast_profiler_update(PROFILER_TIME_MARIO);
+        }
```

Comment:
are these if statements compiled out if fast_profiler_update is a define noop?

Reply:
Yeah, check the header. If the define is disabled, the functions become macros that do nothing, so this compiles into nothing as well.